executor/src/witgen/global_constraints.rs

```diff
@@ -102,15 +103,17 @@ impl<T: FieldElement> RangeConstraintSet<&AlgebraicReference, T> for GlobalConst
     }
 }
 
+type Identity<T> = AnalyzedIdentity<SelectedExpressions<Expression<T>>>;
```

Comment:
Please move this to a more central point. Could be at the root of this crate or even at some exeternal-facing point of the podr_ast::analyzed module.